docs/src/content/docs/book/statements.mdx

```diff
@@ -53,6 +53,29 @@ fun getTimeFromNow(offset: Int): Int {
 }
 ```
 
+If the function does not have an explicit return type, it has an implicit return type of `void`. As such, the `return{:tact}` statement must be empty.
+
+```tact
+extends mutates fun correct(self: Int) {
+    if (self == 42) {
+        return;
+    }
+
+    self = 42;
+}
```

Comment:
need a better example. try to look in for examples in the benchmarks or at least in the end-to-end tests

Reply:
A better example would include an `if` statement with two reachable returns in its branches vs just one reachable return 

also, `throw` should be showcased here

Reply:
It needs to be said the `return`-reachability analysis is not almighty. And show an example where for a human it is more or less obvious a function always returns but the analysis rejects it